src/LocationQuestionAdminRegionComponent.tsx

```diff
@@ -0,0 +1,137 @@
+import { JsonQLQuery } from "jsonql"
+import { ExprCompiler, Schema } from "mwater-expressions"
+import MWaterDataSource from "mwater-expressions/lib/MWaterDataSource"
+import React, { useEffect, useState } from "react"
+
+type LocationQuestionAdminRegionComponentProps = {
+  apiUrl?: string
+  login?: any
+
+  location: { longitude: number; latitude: number; method: string }
```

Comment:
method isn't used?

Reply:
which method?

Reply:
No, I mean the member "method" of the location field.

Reply:
Ah, yes. I just typed it according to the data available in location question answer